### PR TITLE
LRDOCS-4931 Element Sets

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4931, Element Sets article for SME Review.

### CODE REVIEW

File: discover/portal/articles/130-forms/05-element-sets.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Form Element Sets
+
+If you're here looking for information on reusable field sets, you're in the
+right place. We call them Element Sets in the Liferay Forms application because
+these sets include more than just fields: they include the layout and
+configuration of the fields as well. In the future, additional styling elements
+will be available here, too. <!--Is this true?-->.
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/05-element-sets.markdown:7]: Unnecessary dot after comment.

In addition, answering your question written in the comment, you can say that for our readers, once it's planned to add styling elements, like: Fieldset, Separator, etc. 😃

File: discover/portal/articles/130-forms/05-element-sets.markdown [hunk 2]
```diff
@@ -27,26 +27,28 @@ and drop elements onto the form builder palette, configuring fields as you go.
 When you're finished, click *Save*. Element Sets aren't publishable, so there's
 no button for that.
 
-SCREENSHOT
+![Figure 1: Creating Element Sets is just like creating Forms. You just can't publish them.](../../images/forms-element-sets.png)
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/05-element-sets.markdown]: Currently, there is an UI bug which has moved the _Add button_ to the wrong position and, unfortunately, the screenshot is displaying this issue. Can you please circumvent it editing the image? Maybe you can cut a square from the layout background and paste in front of the _Add button_. What do you think about this solution?

**Review by natocesarrego — COMMENTED** [review]
This article is fully reviewed. Please see my notes.

Thanks.

File: discover/portal/articles/130-forms/05-element-sets.markdown (hunk 2 above)
Comment (natocesarrego) [reply discover/portal/articles/130-forms/05-element-sets.markdown]: Just to let you know, this issue will be addressed through the https://issues.liferay.com/browse/LPS-83390. 😉

Comment (rbohl) [reply discover/portal/articles/130-forms/05-element-sets.markdown]: I just cropped it out.

**Pull request merged**